frog/imports/client/Wiki/ModalPageSettings.js

```diff
@@ -158,19 +170,10 @@ class NewPageModal extends React.Component<PropsT, StateT> {
         operatorTypesList.push('op-aggregate');
       }
     }
-
     return (
       <Dialog
```

Comment:
We should use the Modal object and the Modal Controller

Reply:
In this case, we need to have custom styling which would not be possible by the Modal Object